Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Why this file need?

Reply:
sorry, it shouldn't be there. I'll remove it